contrib/node/src/python/pants/contrib/node/tasks/node_paths.py

```diff
@@ -4,10 +4,12 @@
 
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-
 # TODO(John Sirois): UnionProducts? That seems broken though for ranged version constraints,
```

Comment:
This comment should go after the import, as it is specific to `NodePaths` I think.